# poky uses new name to differentiate from modutils, so reflect this new name in source
#
# comment added by Kevin Tian <kevin.tian@intel.com>, 2010-07-21

Upstream-Status: Inappropriate [embedded specific]

--- module-init-tools-3.0-pre10.orig/generate-modprobe.conf
+++ module-init-tools-3.0-pre10/generate-modprobe.conf
@@ -45,12 +45,12 @@
     cp $TESTING_MODPROBE_CONF $MODPROBECONF
 elif [ "$STDIN" = "1" ]; then
     cat > $MODPROBECONF
-elif [ -x /sbin/modprobe.old ]; then
+elif [ -x /sbin/modprobe.24 ]; then
     # In sbin.
-    /sbin/modprobe.old -c > $MODPROBECONF || modprobe_abort
-elif modprobe.old -c >/dev/null 2>&1; then
+    /sbin/modprobe.24 -c > $MODPROBECONF || modprobe_abort
+elif modprobe.24 -c >/dev/null 2>&1; then
     # Somewhere in path.
-    modprobe.old -c > $MODPROBECONF || modprobe_abort
+    modprobe.24 -c > $MODPROBECONF || modprobe_abort
 elif /sbin/modprobe -V 2>/dev/null | grep -q 'modprobe version'; then
     # Running /sbin/modprobe gives old version.
     /sbin/modprobe -c > $MODPROBECONF || modprobe_abort
